storage/s3/src/main/java/io/aiven/kafka/tieredstorage/storage/s3/S3StorageConfig.java

```diff
@@ -48,6 +48,14 @@ public class S3StorageConfig extends AbstractConfig {
         + "By default, empty value means S3 library will auto-detect. "
         + "Amazon S3 uses virtual hosts by default (true), but other S3-compatible backends may differ (e.g. minio).";
 
+    private static final String S3_UPLOAD_MULTI_PART_SIZE_BYTES_CONFIG = "s3.upload.multi.part.size.bytes";
```

Comment:
Maybe just
```suggestion
    private static final String S3_UPLOAD_MULTI_PART_SIZE_BYTES_CONFIG = "s3.upload.part.size.bytes";
```
?

Also I think for consistency with `chunk.size` we should drop `bytes`.

Reply:
Can this lead to a confusion between chunk and part? IMO having multipart in the name is a little easier to understand but I'm fine with both.

Reply:
You're right, good point. `s3.miltipart.upload.part.size` maybe then?